src/App.tsx

```diff
@@ -1,10 +1,33 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useContext, useEffect, useState } from 'react';
+
+import { TodosHeader } from './Components/TodosHeader';
+import { TodosFooter } from './Components/TodosFooter';
+import { Todoslist } from './Components/TodosList';
+import { TodoContext, TodosProvider } from './Components/TodosContext';
 import { UserWarning } from './UserWarning';
+import { ErrorNotification } from './Components/ErrorNotification';
+import { ErrorMessage } from './Enum/ErrorMessage';
+import { client } from './utils/fetchClient';
+import { Todo } from './types/Todo';
 
-const USER_ID = 0;
+const USER_ID = 11360;
+const URL = `/todos?userId=${USER_ID}`;
 
 export const App: React.FC = () => {
+  const { setTodo } = useContext(TodoContext);
+  const [isError, setIsError] = useState<ErrorMessage>(ErrorMessage.NONE);
+
+  useEffect(() => {
+    client.get(URL).then(data => {
```

Comment:
you can use 
<img width="589" alt="image" src="https://github.com/mate-academy/react_todo-app-loading-todos/assets/70220667/8c5d3a7d-b860-49d7-8130-05e0f11623c7">